doc/requests.rst

```diff
@@ -1,54 +0,0 @@
-==========================
-Requests for pull requests
-==========================
-
-We always welcome new feature requests and suggestions, but if you are looking
-to contribute, here are some features that we plan to add soon.
-
-Exponential family distributions
---------------------------------
-
-One prominent next step is to increase the number of distributions
-available. In rough order of importance, we are looking at:
```

Comment:
Should we move this stuff to the wiki? So that it's not lost completely? Perhaps ideas for a GSoC student? :)

Reply:
not high priority; resolving